src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/inventory/inventory-devices/inventory-devices.component.html

```diff
@@ -0,0 +1,42 @@
+<cd-table [data]="filterInDevices"
+          [columns]="columns"
+          identifier="uid"
+          [forceIdentifier]="true"
+          columnMode="flex"
+          [autoReload]="false">
+  <div class="table-filters form-inline"
```

Comment:
What do you think about hiding the table filters when showing the devices filters?
It would fix 2 nitpicks:
- The table filter is breaking into second line, I think all 3 elements should move into a separate line when breaking (Should be fix in a separate PR)
- It feels strange to filter the table with some string and then you Add the filters and it will not show up in the OSD Create form.

The following image exemplifies both issues. If you submit that filters you will not get what you are currently seeing.
![image](https://user-images.githubusercontent.com/399326/69534175-c7b89680-0f68-11ea-9712-abaa1b1d3abd.png)

Reply:
Thanks! i18n issue is fixed and squashed.
I removed the search field in device table, to achieve this I need to add a input parameter to cd-table component, please see https://github.com/ceph/ceph/pull/30921/commits/74194c48eaaaa17e6d659decd53299a45f98567f